portality/static/widget/fq_widget_depends_compiled.js

```diff
@@ -1026,12 +1026,12 @@ function dateHistogramShowLess(options, context, facet) {
 }
 
 var licenceMap = {
-    "CC BY" : ["https://doaj.org/static/doaj/images/cc/by.png", "http://creativecommons.org/licenses/by/3.0/"],
-    "CC BY-NC" : ["https://doaj.org/static/doaj/images/cc/by-nc.png", "http://creativecommons.org/licenses/by-nc/3.0/"],
-    "CC BY-NC-ND" : ["https://doaj.org/static/doaj/images/cc/by-nc-nd.png", "http://creativecommons.org/licenses/by-nc-nd/3.0/"],
-    "CC BY-NC-SA" : ["https://doaj.org/static/doaj/images/cc/by-nc-sa.png", "http://creativecommons.org/licenses/by-nc-sa/3.0/"],
-    "CC BY-ND" : ["https://doaj.org/static/doaj/images/cc/by-nd.png", "http://creativecommons.org/licenses/by-nd/3.0/"],
-    "CC BY-SA" : ["https://doaj.org/static/doaj/images/cc/by-sa.png", "http://creativecommons.org/licenses/by-sa/3.0/"]
```

Comment:
Cool, sounds reasonable then. I do have one question though ... do we also need to make these changes to the normal JS files besides the compiled file? The compiled file is really supposed to be just a reflection of the collection of our normal multi-file JS code, though I appreciate it's the thing that will actually get loaded for the widget. I'm apprehensive to let the separate-file-codebase and this big file diverge, or rather I give priority to the separate files since I can pretty much mechanically regenerate this.